src/App.tsx

```diff
@@ -2,36 +2,83 @@ import ChatBot, { Flow } from "react-chatbotify";
 
 import RcbPlugin from "./factory/RcbPluginFactory";
 import { InputValidatorBlock } from "./types/InputValidatorBlock";
+import { validateFile } from "./utils/validateFile";
 
 const App = () => {
-	// initialize example plugin
-	const plugins = [RcbPlugin()];
-
-	// example flow for testing
-	const flow: Flow = {
-		start: {
-			message: "Hey there, please enter your age!",
-			path: "try_again",
-			validateInput: (userInput: string) => {
-				if (typeof userInput === "string" && !Number.isNaN(Number(userInput))) {
-					return {success: true};
-				}
-				return {success: false, promptContent: "Age must be a number!", promptDuration: 3000, promptType: "error", highlightTextArea: true};
-			}
-		} as InputValidatorBlock,
-		try_again : {
-			message: "Nice, you passed the input validation!",
-			path: "start",
-		}
-	}
-
-	return (
-		<ChatBot
-			id="chatbot-id"
-			plugins={plugins}
-			flow={flow}
-		></ChatBot>
-	);
-}
-
-export default App;
\ No newline at end of file
+  // Initialize the plugin
+  const plugins = [RcbPlugin()];
+
+  // Example flow for testing
+  const flow: Flow = {
+    start: {
+      message: "Hey there! Please enter your age.",
+      path: "age_validation",
+      validateTextInput: (userInput?: string) => {
+        if (userInput && !Number.isNaN(Number(userInput))) {
+          return { success: true };
+        }
+        return {
+          success: false,
+          promptContent: "Age must be a number!",
+          promptDuration: 3000,
+          promptType: "error",
+          highlightTextArea: true,
+        };
+      },
+    } as InputValidatorBlock,
+
+    age_validation: {
+      message:
+        "Great! Now please upload a profile picture (JPEG or PNG) or provide a URL.",
+      path: "file_upload_validation",
+      chatDisabled: true, // Set to true if you want to disable text input
+      validateTextInput: (userInput?: string) => {
+        console.log("validateTextInput called with userInput:", userInput);
+
+        if (userInput && userInput.trim().length > 0) {
+          // Optionally, validate if the input is a valid URL
+          // For simplicity, we'll accept any non-empty text
+          return { success: true };
+        }
+
+        return {
+          success: false,
+          promptContent:
+            "Please provide a valid URL or upload a file.",
+          promptDuration: 3000,
+          promptType: "error",
+        };
+      },

```

Comment:
No need for `validateTextInput` here, since `chatDisabled` is already set to true.